lime/app/Config.hx

```diff
@@ -27,5 +27,6 @@ typedef Config = {
 	@:optional var version:String;
 	@:optional var vsync:Bool;
 	@:optional var width:Int;
+	@:optional var highdpi:Bool;
```

Comment:
I felt a bit weird, since DPI is an acronym, if I do highDPI (Which I did), then the internal flag becomes HIGH_D_P_I, which sucks. And we have "vsync" which is also not capitalized, so.